tests/kafkatest/services/security/security_config.py

```diff
@@ -166,10 +167,22 @@ def has_sasl_kerberos(self):
 
     @property
     def kafka_opts(self):
+        if SecurityConfig.ZK_JAAS_LOGIN:
+            return "\"-Djava.security.auth.login.config=%s\"" % SecurityConfig.ZK_JAAS_LOGIN
```

Comment:
Also - does this work if sasl is specified for for broker to client connections as well as broker to zookeeper?